cilium-health/launch/endpoint.go

```diff
@@ -222,13 +223,19 @@ type EndpointAdder interface {
 	AddEndpoint(owner regeneration.Owner, ep *endpoint.Endpoint, reason string) error
 }
 
+type identityAllocator interface {
+	WaitForInitialGlobalIdentities(context.Context) error
+	AllocateIdentity(context.Context, labels.Labels, bool) (*identity.Identity, bool, error)
+	Release(context.Context, *identity.Identity) (released bool, err error)
+}
```

Comment:
Maybe this interface should be defined in `pkg/identity` instead of defining it multiple times in different packages?

Reply:
Will be fixed in next upload!